include/evaluators/blas1_tree_evaluator.hpp

```diff
@@ -0,0 +1,152 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) 2016 Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename blas1_tree_evaluator.hpp
+ *
+ **************************************************************************/
+
+#ifndef BLAS1_TREE_EVALUATOR_HPP_CDFFQDGR
+#define BLAS1_TREE_EVALUATOR_HPP_CDFFQDGR
```

Comment:
So, the random thing is generated by my IDE, I will remove it. The reason to use it was because I was copying operations/ files into evaluators/, and the header guards would be the same. We can stick to the convention if we agree on one, I don't mind.